File: 1914.py
casos_teste = int(input())
for i in range(casos_teste):
    jogadores = input().split(" ")
    numeros = input().split(" ")
    soma = int(numeros[0]) + int(numeros[1])
    resultado = "PAR" if (soma % 2 == 0) else "IMPAR"
    vencedor = jogadores[0] if (jogadores[1] == resultado) else jogadores[2]
    print(vencedor)

File: 1914.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var casos_teste = Number(lines.shift().trim());
for (var i = 0; i < casos_teste; i++) {
    var jogadores = lines.shift().trim().split(" ");
    var numeros = lines.shift().trim().split(" ");
    var soma = Number(numeros[0]) + Number(numeros[1]);
    var resultado, vencedor;
    resultado = (soma % 2 === 0) ? "PAR" : "IMPAR" ;
    vencedor = (jogadores[1] === resultado) ? jogadores[0] : jogadores[2] ;
    console.log(vencedor);
}
